frontend/src/app/core/services/auth.service.ts

```diff
@@ -43,10 +43,10 @@ export class AuthService {
     }
 
     public isAuthorized() {
-        return this.getUserToken() && this.getUserInfo();
+        return this.getUserToken() ? this.getUserInfo() : null;
```

Comment:
I think this method cannot be used to get user. I'd recommend create new function, for example, getUser